src/other/broadcasting.jl

```diff
@@ -78,7 +78,8 @@ struct LazyNewColDataFrame{T}
     col::T
 end
 
-# we allow LazyNewColDataFrame only for data frames with at least one column
+# we allow LazyNewColDataFrame also for data frames
+# that are empty, ie. `nrow(df) == 0`; in this case we create a 0 length column
```

Comment:
This comment looks weird when you don't know the history: why wouldn't it be allowed for 0-row data frames? I'd just remove it, or put it elsewhere.

Reply:
fixed